autocomplete/definitions/global/mwse/log.lua

```diff
@@ -5,6 +5,6 @@ return {
 The message accepts formatting and additional parameters matching string.format's usage.]],
 	arguments = {
 		{ name = "message", type = "string" },
-		{ name = "formatValues", type = "variadic", optional = true }
+		{ name="...", type = "any", description="Formatting arguments. These are passed to `string.format`.", optional = true }
```

Comment:
Small nit: can you put a space before and after operators?

Reply:
good catches. ill address both those things